src/components/SpotlightCard/SpotlightCard.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState, useEffect } from "react";
+import "./SpotlightCard.scss";
+import TextLink from "../TextLink";
+
+export interface SpotlightCardProps {
+    title: string;
+    imageSource?: string;
+    imageAltText?: string;
+    subtitle?: string;
+    description?: string;
+    textLink?: string;
+    halfWidth?: boolean;
+    // TODO add CTA button when access to CFA repo is granted
+}
+
+const SpotlightCard = (props: SpotlightCardProps) => {
+    const [expanded, setExpanded] = useState(false);
+    const [description, setDescription] =  useState('');
+    const [showReadToggle, setShowReadToggle] = useState(false);
+    const [readToggleText, setReadToggleText] = useState('');
+
+    const textLinkText = 'Meet the Leadership Team';
+    const substringLength = 385;
+    const mobile = window.innerHeight <= 767 && window.innerWidth <= 767;
+    const halfWidth = props.halfWidth !== undefined && !mobile ? props.halfWidth : false;
+
+    useEffect(() => {
+        if(props.description !== undefined){
+            let description = '';
+            let showReadToggle = false;
+            let readToggleText = '';
+
+            if(expanded) {
+                description = props.description;
+                if(props.description.length > substringLength){
+                    showReadToggle = true;
+                    readToggleText = 'Read Less';
+                }
+            } else {
+                description = props.description.substring(0,substringLength);
+                if(props.description.length > substringLength){
+                    description += '...';
+                    showReadToggle = true;
+                    readToggleText = 'Read More';
+                }
+            }
+
+            setDescription(description);
+            setShowReadToggle(showReadToggle);
+            setReadToggleText(readToggleText);
+        }
+    }, [expanded])
+    
+    return (
+      <div className= {'spotlight-card mt mb ' + (halfWidth ? 'spotlight-card-half-width' : '')}>
+        <div className={'spotlight-card__img-container ' + (halfWidth ? 'spotlight-card__img-container-half-width' : '')}>
```

Comment:
We have this round image with dotted border in another component called the Team Member Profile Image. Can you try to reuse that?

Reply:
We cannot dynamically size that component, so I just ended up remaking it here. If we want to use should, we should extend that component so that it can be dynamically resized, would you like me to do that?

Reply:
Using the TeamMemberProfileImage, but it does not allow us to pass in an imageAltText, just FYI (I think we should probably add that in)